modules/010-deckhouse-flow-schema/enabled

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright 2021 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /deckhouse/shell_lib.sh
+
+function __main__() {
+  enabled::disable_module_if_cluster_is_not_bootstraped
```

Comment:
It is unnecessary; removing the script (as we did for the priority-class module) is better.

Reply:
removed